P0009/P0009.bs

```diff
@@ -1797,7 +1903,7 @@ Table �: Accessor policy requirements
 <tr>
   <td>`A::offset_policy`</td>
   <td></td>
-  <td>Accessor policy which can access a pointer that has been return by `a.offset(p,i)`.</td>
+  <td>Accessor policy which can access a pointer that has been returned by `a.offset(p,i)`.</td>
 </tr>
 <tr>
   <td>`a.decay(p)`</td>
```

Comment:
this is another relaxation point, only required under certain circumstances; returns a span